### PR TITLE
Add suse tests to pipeline

### PR DESCRIPTION
**Pull request by kahowell**
Feel free to review, but not appropriate to merge until the suse branches are all merged in subscription-manager.

### CODE REVIEW

**Comment by kahowell** [comment]
104f28c..326cc32: rebase against master

**Comment by Lorquas** [comment]
326cc32e830cd14c50c449bb978cecffeae6a810 ..  5347b1bf2f93a04472597c249ccfb4b9d90af96a : rebase against master

**Review by Lorquas — APPROVED** [review]
looks good to me as is.

Is there any way to add something like this to simplify code since we're calling it all over the place?

```groovy
def smartGithubNotify = {ref, status,  context,  targetUrl ->
  String desc = ""
  switch (status) {
    case 'PENDING':
      desc = PENDING_MESSAGE
      break
    case 'SUCCESS':
      desc = SUCCESS_MESSAGE
      break
    case 'FAILURE':
      desc = FAILURE_MESSAGE
      break
  }

  ref.with(
    githubNotify(
        account: GITHUB_ACCOUNT,
        repo: GITHUB_REPO,
        credentialsId: GITHUB_CREDENTIALS_ID,
        sha: GITHUB_COMMIT,
        status: status,
        description: desc,
        context: context,
        targetUrl: targetUrl))
 }
```
calling with:
```groovy
smartGithubNotify(
              this,
              status: 'PENDING',
              context: 'python-rhsm-python3',
              targetUrl: BUILD_URL)
```

or does that run into the crazy pipline closures problems like before?

**Comment by kahowell** [comment]
Changes:
 - Refactor the subman pipeline into the patterns developed for the
candlepin pipeline.
 - Remove sles11 jobs, since right now sles11 builds are blocked by
python-six not being available, along with other various issues.
 - Make a few fix to the suse jobs for issues found in smoketesting.
 - s/Candlepin Performance/CandlepinPerformance

File: jobs/submanPipeline.groovy [hunk 1]
```diff
@@ -1,131 +1,131 @@
...
+    try {
+        githubNotify(account: GITHUB_ACCOUNT, repo: GITHUB_REPO, credentialsId: GITHUB_CREDENTIALS_ID, sha: GITHUB_COMMIT,
+                status: args.status, description: description, context: args.context, targetUrl: args.targetUrl)
+    } catch (e) {
+        echo "Unable to update GitHub status (commit: ${GITHUB_COMMIT}, status: ${args.status}, context: ${args.context})"
+    }
+}
+
+def buildWithNotifications(Map args) {
+    boolean enabled = Jenkins.instance.getItem(args.job, currentBuild.rawBuild.parent, Item.class).buildable
```
Comment (cnsnyder) [inline jobs/submanPipeline.groovy]: Perhaps surround this in a try catch. I've recently seen with the candlepin pipeline somewhat cryptic failure messages that arise due to changes to the job name (Changed one job from "Candlepin Performance" to "CandlepinPerformance").

Non-nacking, but please consider!

**Review by cnsnyder — CHANGES_REQUESTED** [review]

**Comment by kahowell** [comment]
2dcd69d..15f606f: skip job if it can't be found

File: jobs/submanPipeline.groovy (hunk 1 above)
Comment (kahowell) [reply jobs/submanPipeline.groovy]: @cnsnyder done!

**Review by cnsnyder — APPROVED** [review]
LGTM!

**Pull request merged**